public/app/features/alerting/unified/components/amnotifications/AmNotificationsGroupHeader.tsx

```diff
@@ -21,11 +20,14 @@ export const AmNotificationsGroupHeader = ({ group }: Props) => {
   }, {} as Record<AlertState, number>);
 
   return (
-    <div className={styles.summary}>
-      {`${total} alerts: `}
+    <div>
+      {`${total} alert${total > 1 ? 's' : ''}: `}
```

Comment:
I thnk we have https://www.npmjs.com/package/pluralize installed that you could use